artifacts/index.ts

```diff
@@ -2,23 +2,16 @@ export { AddressToAddressWhiteList } from "./ts/AddressToAddressWhiteList";
 export { AuctionMock } from "./ts/AuctionMock";
 export { BadCTokenMock } from "./ts/BadCTokenMock";
 export { BadTokenMock } from "./ts/BadTokenMock";
-export { Bytes32LibraryMock } from "./ts/Bytes32LibraryMock";
```

Comment:
Why remove all of these?

Reply:
They aren't used elsewhere, like I thought they would be when I was exporting helpers. So just cleaning up.